gtfs/spec/en/reference.md

```diff
@@ -248,16 +250,16 @@ File: **Conditionally Required**
 
 |  Field Name | Type | Presence | Description |
 |  ------ | ------ | ------ | ------ |
-|  `service_id` | ID referencing `calendar.service_id` or ID | **Required** | Identifies a set of dates when a service exception occurs for one or more routes. Each (`service_id`, `date`) pair can only appear once in [calendar_dates.txt](#calendar_datestxt) if using [calendar.txt](#calendartxt) and [calendar_dates.txt](#calendar_datestxt) in conjunction. If a `service_id` value appears in both [calendar.txt](#calendartxt) and [calendar_dates.txt](#calendar_datestxt), the information in [calendar_dates.txt](#calendardatestxt) modifies the service information specified in [calendar.txt](#calendartxt). |
+|  `service_id` | ID referencing `calendar.service_id` or ID | **Required** | Identifies a set of dates when a service exception occurs for one or more routes. Each (`service_id`, `date`) pair may only appear once in [calendar_dates.txt](#calendar_datestxt) if using [calendar.txt](#calendartxt) and [calendar_dates.txt](#calendar_datestxt) in conjunction. If a `service_id` value appears in both [calendar.txt](#calendartxt) and [calendar_dates.txt](#calendar_datestxt), the information in [calendar_dates.txt](#calendardatestxt) modifies the service information specified in [calendar.txt](#calendartxt). |
 |  `date` | Date | **Required** | Date when service exception occurs. |
-|  `exception_type` | Enum | **Required** | Indicates whether service is available on the date specified in the date field. Valid options are:<br><br> `1` - Service has been added for the specified date.<br>`2` - Service has been removed for the specified date.<hr>*Example: Suppose a route has one set of trips available on holidays and another set of trips available on all other days. One `service_id` could correspond to the regular service schedule and another `service_id` could correspond to the holiday schedule. For a particular holiday, the [calendar_dates.txt](#calendar_datestxt) file could be used to add the holiday to the holiday `service_id` and to remove the holiday from the regular `service_id` schedule.* |
+|  `exception_type` | Enum | **Required** | Indicates whether service is available on the date specified in the date field. Valid options are:<br><br> `1` - Service has been added for the specified date.<br>`2` - Service has been removed for the specified date.<hr>*Example: Suppose a route has one set of trips available on holidays and another set of trips available on all other days. One `service_id` could correspond to the regular service schedule and another `service_id` could correspond to the holiday schedule. For a particular holiday, the [calendar_dates.txt](#calendar_datestxt) file can be used to add the holiday to the holiday `service_id` and to remove the holiday from the regular `service_id` schedule.* |
```

Comment:
Shouldn't it be "may" instead of "can"?

Reply:
This section is tricky because it's describing a hypothetical usage of the fields via an example. I am tending towards "could" being okay here and even reverting the final "can" back to "could" to be consistent with the tense. 

@lionel-nj Put it elegantly. As a quick reference:
- Can - Ability to do something (i.e., scenarios, usage)
     - Example: riders ability do to things -> can; e.g. _“riders can reach a customer service…”_
- May - Permission/possibility (i.e., spec)
     - Example: restrictions from the specification -> may; e.g. _“The `stop_code` may be the same as `stop_id` if it is public facing.”_

Reply:
Please see a revised suggestion in the "Files changed" tab. Feel free to provide a second review.